src/environments/environment.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
     production: false,
     langKey: 'lang',
-    defaultLang: 'en'
+    defaultLang: 'en',
+    api_url: 'http://localhost:3000'
```

Comment:
we shouldn't hardcode localhost as our route
what will be when we will deploy our app to some domain?